jekyll/_cci2/server-3-operator-metrics-and-monitoring.adoc

```diff
@@ -27,22 +27,9 @@ cases. Hence, features around local observability are provided on an as-is basis
 flexibility and extensibility of the system, should you feel that the functionality provided is not sufficient for your
```

Comment:
maybe a @nathanfishcircleci questions, but I don't think we want to be encouraging customizations around Prometheus just yet.  There is not any reasonable way for the customer to go about it.  I would probably remove most of this section

Reply:
Great catch. I think this should probably be specific to Telegraf customizations

Reply:
done